### ISSUE TITLE
Can't import pluginlibrary into another project

### ISSUE THREAD

**Issue by varumora-everywoah** [ricohapi/theta-plugin-sdk#3]
I'm trying to import pluginlibrary into my own project (even if it is an empty one) and I get this error:

Element: class org.jetbrains.plugins.groovy.lang.psi.impl.GroovyFileImpl because: different providers: SingleRootFileViewProvider{myVirtualFile=file:///Users/varu/Documents/Android/BluetoothConnectionTheta/settings.gradle, content=VirtualFileContent{size=16}}(7b00ae08); SingleRootFileViewProvider{myVirtualFile=file:///Users/varu/Documents/Android/BluetoothConnectionTheta/settings.gradle, content=VirtualFileContent{size=16}}(5f3af20d)

The folder is copied to the directory but is not shown in Android Studio

**Comment by varumora-everywoah** [comment]
Making .aar and importing it worked though

**Comment by shohara** [comment]
I've tried to make a new project and import the pluginlibrary by selecting "File"-"New"-"Import Module..." in Android Studio 3.1.3, but I had a same issue. Then I obeyed the [hint](https://stackoverflow.com/questions/49811283/android-studio-3-1-1-unable-to-add-module-import-gradle-project). 

0. Import "pluginlibrary" in the SDK by selecting "File"-"New"-"Import Module..."
1. Add a text "include ':app'**_, ':pluginlibrary'_**" in "settings.gradle" file 
2. Sync by selecting "File"-"Sync Project with Gradle Files". 

After that I could import pluginlibrary to my project. I hope this helps you.
